test/unit_test/instantiate_fft_tests.hpp

```diff
@@ -241,6 +241,13 @@ INSTANTIATE_TEST_SUITE_P(workItemStridedOOPInOrder, FFTTest,
                              ::testing::Values(layout_params{{3}, {4}, {7}}, layout_params{{8}, {11}, {2}},
                                                layout_params{{9}, {3}, {4}, 30, 40}))),
                          test_params_print());
+INSTANTIATE_TEST_SUITE_P(
+    SubgroupStridedOOPInOrder, FFTTest,
+    ::testing::ConvertGenerator<layout_param_tuple>(::testing::Combine(
+        oop_unpacked_unpacked_layout, both_directions, complex_storages, ::testing::Values(1, 3, 33000ul),
```

Comment:
What does the smaller batch size test that the larger batch does not?

Reply:
I'm not 100% sure tbh. Potentially instances when only part of a subgroup is utilized.

Reply:
Seems unlikely to me - your sizes are 64, 96, and 128, so will have a power-of-2 factor equal to the subgroup size. None of the stragglers mentioned in the other comment.

I like the idea that tests all have a purpose, rather than thinking that test coverage must be good because we instantiate lots of cases.

Reply:
larger variety of sizes added here: https://github.com/codeplaysoftware/portFFT/pull/142/commits/cca8e4c631d9818beb4a9484f86775c329eedab2

I think I see the point you're making, we probably don't always need to run the small batch sizes. I think is is also valuable to have batch sizes that are not multiples of the number of ffts per subgroup/workgroup. You could do that with a large size or a small size and I think it would be fine.